packages/ui/src/ui/utils/acl/acl-api.ts

```diff
@@ -246,7 +248,17 @@ export function requestPermissions(params: RequestPermissionParams): Promise<Upd
     };
     return yt.v3.executeBatch(batchParams).then((results: BatchResultsItem<unknown>[]) => {
         const error = getBatchError(results, 'Failed to request permissions');
+
         if (error) {
+            toaster.add({
+                name: 'get operation',
+                autoHiding: false,
+                theme: 'danger',
+                title: 'Failed to request permissions',
+                content: error.message,
+                actions: [{label: ' view', onClick: () => showErrorPopup(error)}],
+            });
```

Comment:
We prefer to display error as close to its data as possible. Why aren't such errors displayed in their dialog?